core/services/ocr2/plugins/ocr2keeper/evm21/encoding/packer.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg/v3/types"
+	"github.com/smartcontractkit/spike-stream-lookup/mercury"
```

Comment:
spike-stream-lookup strikes me as an odd name. why not `stream-lookup`?

Reply:
It's an intentionally bad name; none of this will be merged and that repo will eventually be nuked too. Naming in this way helps indicate that the repo was created as part of a spike/investigation, and shouldn't be considered production ready.